pkgs/development/php-packages/deployer/default.nix

```diff
@@ -11,12 +11,17 @@ mkDerivation rec {
 
   dontUnpack = true;
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
 
   installPhase = ''
     mkdir -p $out/bin
     install -D $src $out/libexec/deployer/deployer.phar
     makeWrapper ${php}/bin/php $out/bin/dep --add-flags "$out/libexec/deployer/deployer.phar"
+
+    # fish support currently broken: https://github.com/deployphp/deployer/issues/2527
+    installShellCompletion --cmd dep \
+      --bash <($out/bin/dep autocomplete --install) \
+      --zsh <($out/bin/dep autocomplete --install)
```

Comment:
Are those commands supposed to be the same?

Reply:
Yes. This is listed in documentation... even for `fish` completion. Which explains why `fish` completion doesn't work :disappointed: I only tested the `bash` completion. I have never really dabbled around with `zsh` to know if it is realistic to expect this to actually work or not...

Reply:
bash scripts mostly work under bash so I just trust upstream that it works for now.